posthog/migrations/0091_messagingrecord.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.0.7 on 2020-10-22 11:33
+
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0090_org_live"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="MessagingRecord",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=posthog.models.utils.UUIDT, editable=False, primary_key=True, serialize=False
+                    ),
+                ),
+                ("email_hash", models.CharField(max_length=1024)),
```

Comment:
I think Postgres is smart enough not to overallocate space for varchars, but this `max_length` seems very very generous considering that SHA-256 hashes are always 64 chars in hexadecimal representation AND email addresses on our Django model are also restricted to 254 chars.

Reply:
Right! Forgot to change this when I decided to go with a hash instead of the email. Will change that at the next opportunity.